libparsec/crates/types/tests/unit/manifest.rs

```diff
@@ -39,3 +44,259 @@ fn invalid_deserialize_data(#[case] data: &[u8], #[case] error: DataError) {
 
     assert_eq!(manifest, Err(error));
 }
+
+#[rstest]
+fn dump_load(alice: &Device) {
+    // The blobs are generated with the expected one, to lock the behavior
```

Comment:
I'd rather have a description of the content of the payload and of from what version it has been generated

see what is done for the protocol crate:
```rust
    // Generated from Python implementation (Parsec v2.6.0+dev)
    // Content:
    //   cmd: "ping"
    //   ping: "ping"
```